editor/plugins/node_3d_editor_plugin.cpp

```diff
@@ -1488,7 +1488,9 @@ Transform3D Node3DEditorViewport::_compute_transform(TransformMode p_mode, const
 }
 
 void Node3DEditorViewport::_surface_mouse_enter() {
-	if (!surface->has_focus() && (!get_viewport()->gui_get_focus_owner() || !get_viewport()->gui_get_focus_owner()->is_text_field())) {
+	Input::MouseMode mouse_mode = Input::get_singleton()->get_mouse_mode();
+
+	if (mouse_mode != Input::MouseMode::MOUSE_MODE_CAPTURED && (!surface->has_focus() && (!get_viewport()->gui_get_focus_owner() || !get_viewport()->gui_get_focus_owner()->is_text_field()))) {
```

Comment:
```suggestion
	if (mouse_mode != Input::MouseMode::MOUSE_MODE_CAPTURED && (!surface->has_focus() && !get_viewport()->gui_get_focus_owner() || !get_viewport()->gui_get_focus_owner()->is_text_field())) {
```

Reply:
Ugh hang on. That's probably not what I should have done...

Reply:
What? The parentheses are redundant

Reply:
I meant clicking commit suggestion on Github. Unless that's proper protocol? I know you all prefer the commits squashed.

Reply:
You can squash after it's approved 🙂

Reply:
Gotcha, thanks.